web/src/pages/SignupPage/SignupPage.tsx

```diff
@@ -52,7 +52,9 @@ const SignupPage = () => {
         <Toaster toastOptions={{ className: 'rw-toast', duration: 6000 }} />
         <header className='flex items-center justify-start w-full h-16 px-4'>
```

Comment:
The `header` is being used on two pages that are almost identical — looks like a good place to implement a Layout component (DRY)